### ISSUE TITLE
Add Red Zeppelin 5 kit

### ISSUE THREAD

**Issue by SpotlightKid** [x42/avldrums.lv2#14]
The [current distribution](http://bandshed.net/sounds/sf2/AVL-Drumkits-1.1-fix-SF2.zip) of AVLDrums also contains a version of the Red Zeppelin Kit with samples of an additional floor tom (instead of the 16" floor tom edge, I believe), called "Red Zeppelin 5".

It would be nice to add variant of the LV2 plugin for this kit too.

This would make a new UI necessary (new picture and new hit map) and maybe even a few changes to the structure of the plugin to support multiple different midnams and text overlays in the UI.

**Comment by x42** [comment]
It'd likely need to be a new plugin (new URI), too.
Picture has to come from Glen (he photographed the actual kit).

Keep in mind that the .sf2 was significantly modified from the original. Panning matches the one on the image, leading silence has been cut, velocity-layered samples were aligned and exclusive groups were added.

I may sit this one out until Glen finishes his percussion-set and then schedule a long-week-end of sample-lib overhaul for the whole thing.

I also regret not using a standard-MIDI mapping for the main kit -- and adding extra samples outside that default range. if an URI update is due, I might re-do the mapping, too.

**Comment by SpotlightKid** [comment]
> Picture has to come from Glen (he photographed the actual kit).

Yep, though I have a mind to try to render a picture from a nice Blender 3D model of a drumkit. Need to find good free model first and also for the various percussion bits.

> Keep in mind that the .sf2 was significantly modified from the original.

Yes, I read the readme in the `sf2`folder. Did you do these edits manually? It's easy enough to export the samples with Polyphone and edit and re-import them. A comparison of the samples of original Red Zeppelin 4 and 5 kits might show that the samples are the same  except for the 18" floor tom. Then only those samples would need to be edited as well.

> I also regret not using a standard-MIDI mapping for the main kit -- and adding extra samples outside that default range.

BTW, what about the Roto drums?

**Comment by x42** [comment]
> BTW, what about the Roto drums?

Intentionally left off, since they're not on the picture. upstream (Glen) suggested to not include them, anyway.

**Comment by SpotlightKid** [comment]
> I may sit this one out [...] overhaul for the whole thing.

I you do that, I think it would good to put the note-mappping and the midnam in an external file (for each drumkit), so it can be replaced more easily.

**Comment by x42** [comment]
If you want to mod the kits, use drumgizmo or the .sfz.

The main idea behind the avldrums plugin is to have a **consistent** "works out of the box" experience.

Otherwise it's already easy to just load the .sf2 in fluidsynth along with a custom midnam.

**Comment by SpotlightKid** [comment]
> Otherwise it's already easy to just load the .sf2 in fluidsynth

Except that setting up multi-channel is a hassle and not as efficient.

**Comment by x42** [comment]
More multi-channel will be tricky anyway. At most there are 16 individual outputs, since
the plugin uses MIDI-channels for this under the hood.

**Comment by x42** [comment]
OOPS. sorry that commit should have mentioned #16
